### ISSUE TITLE
Accessibility: try-retro-notebooks

### ISSUE THREAD

**Issue by Bennyonye** [jupyterhub/outreachy#75]
### What page is this for?

https://jupyter.org/try-jupyter/retro/notebooks/

### WAVE accessibility report

![Screenshot 2022-10-15 132736](https://user-images.githubusercontent.com/98492400/196040830-56dba9f3-d653-4444-8f00-151d4e778930.jpg)

2 Errors
Missing or uninformative page title
Language missing or invalid

0 Contrast Error

1 Alerts
No page regions

0 Features

1 Structural Element
Heading level 1

0 ARIA

### The HTML element to be changed

```html
1. <title> the page title is missing.
2. <html> without attribute `lang`
3. No page regions
4. Heading level 1
```

### Your proposed HTML (or CSS), after change

```html
1. include the title i.e. 
    <title> JupyterLite Retro - Notebook </title>
2. include HTML lang attribute: i.e. 
     <html lang="en"></html>
3. Since we have contents in the page, the structure should be placed in HTML elements so that it can be 
   identified by the web browser. i.e.
     <main role="main" aria-label="page content">
         <h2 id="region-heading">
           <!-- region content -->
         </h2>
       </main>

                or

     <div role="region" aria-label="page content">
         <h2 id="region-heading">
           <!-- region content -->
         </h2>
     </div>
4. An <h1> element is present; therefore, the page outline is structured correctly.
```

### More info

1. When a screen reader loads a web page, the text within the 'title' is the first thing it sees. If the title does not describe what is on the page, readers who rely on HTML markup will have a difficult time determining which page they are on.  

> The webpage 'title' is added to the page 'head' to assist readers in understanding the specific page they are accessing.

2. The absence of HTML lang can lead to a failure in displaying the local version of the page and if HTML lang is not set, the reading software uses the default language, which might cause a failure in displaying content.  

> Adding the lang attribute to the html tag and choosing a valid value of the lang attribute enable easier reading.

3. Landmark regions are used by screen readers to make it easier to navigate the content on the page by defining the structure of the page. We can do this either using HTML5 landmark elements or <div> elements with aria roles. We should prefer HTML5 to region roles when possible. The region role should be reserved for areas of content that are important enough that visitors would want to easily navigate to them and have them displayed in a page summary. A region role is a more general phrase that should be used only if one of the other landmark roles, such as banner, main, content info, complimentary, or navigation, does not correctly represent the portion that needs to be marked.

4. The level 1 heading is the most important header in the document, It should be rendered more prominently than any other header. It is usually used to indicate the title of the document often it has the same contents as the [TITLE], although this is not required and not always a good idea. The title should be useful out of context (for example, in a bookmarks file) but the level 1 heading is only used inside the document.

The optional ALIGN attribute controls the horizontal alignment of the header. It can be LEFT, CENTER or RIGHT.

**Comment by Chimatara** [comment]
Hi @Bennyonye have you solved the issue?

**Comment by minrk** [comment]
It loosk like this is a page that didn't load in WAVE, and the evaluated HTML is coming from WAVE itself. Perhaps another page would work better?

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
